### ISSUE TITLE
Crafting helmet with probe does not preserve enchantments.

### ISSUE THREAD

**Issue by Roxforbraynz** [McJty/TheOneProbe#29]
http://imgur.com/a/cnX4C

I want to put a probe on my well-enchanted mending helmet in survival, but this prevents me from doing so.

**Comment by McJty** [comment]
Yes, I know about this. I think you enchant it afterwards though. I'll fix this after I return from vacation

**Comment by Roxforbraynz** [comment]
Alright then. I didn't see it in the issue tracker at all, so I posted it. Enjoy your vacation!   :)
